packages/plugin-ext/src/main/browser/view/tree-views-main.tsx

```diff
@@ -218,9 +226,14 @@ export class TreeViewWidget extends TreeWidget {
         @inject(TreeProps) readonly treeProps: TreeProps,
         @inject(TreeModel) readonly model: TreeModel,
         @inject(ContextMenuRenderer) readonly contextMenuRenderer: ContextMenuRenderer,
-        @inject(TreeViewDataProviderMain) readonly dataProvider: TreeViewDataProviderMain) {
+        @inject(TreeViewDataProviderMain) readonly dataProvider: TreeViewDataProviderMain,
+        @inject(SelectionService) readonly selectionService: SelectionService) {
 
         super(treeProps, model, contextMenuRenderer);
```

Comment:
if a widget destroyed, selection should be removed, like here https://github.com/theia-ide/theia/blob/bb6700b60dfc1951df98844bdbb2e072f1e01716/packages/navigator/src/browser/navigator-widget.tsx#L87-L91

Reply:
Fixed